packages/material-ui/src/SelectWidget/SelectWidget.tsx

```diff
@@ -74,11 +80,11 @@ export default function SelectWidget<
       select // Apply this and the following props after the potential overrides defined in textFieldProps
       InputLabelProps={{
         ...textFieldProps.InputLabelProps,
-        shrink: true,
+        shrink: !_isEmpty(),
       }}
       SelectProps={{
         ...textFieldProps.SelectProps,
-        multiple: typeof multiple === "undefined" ? false : multiple,
+        multiple: multiple,
```

Comment:
You can use the shortcut
```suggestion 
        multiple,
```